src/Service/Notification/EntryCommentNotificationManager.php

```diff
@@ -56,6 +56,10 @@ public function __construct(
     // @todo check if author is on the block list
     public function sendCreated(ContentInterface $subject): void
     {
+        if ($subject->user->isBanned || $subject->user->isTrashed()) {
```

Comment:
I'd add a deleted and suspended and everything else check :D

Reply:
done, `isTrashed()` checks the visibility trait for suspended users. i also added `isSoftDeleted()` since it's there...